src/components/molecules/SearchAutocomplete/interface.ts

```diff
@@ -19,6 +19,16 @@ export interface InputStyleProps {
   showSuggestions: boolean;
 }
 
+export interface InputWithTagsProps {
+  onRemoveTag: () => void;
+  text: string;
+  setSuggestionType: (type: string) => void;
```

Comment:
I'd keep the functions at the end, but it's just a nitpick and a matter of style. ;)